exporters/trace/src/main/java/com/google/cloud/opentelemetry/trace/MockCloudTraceClient.java

```diff
@@ -0,0 +1,34 @@
+package com.google.cloud.opentelemetry.trace;
+
+import com.google.devtools.cloudtrace.v2.BatchWriteSpansRequest;
+import com.google.devtools.cloudtrace.v2.ProjectName;
+import com.google.devtools.cloudtrace.v2.Span;
+import com.google.devtools.cloudtrace.v2.TraceServiceGrpc;
+import io.grpc.Channel;
+import io.grpc.ManagedChannelBuilder;
+
+import java.util.List;
+
+// A simplified version of TraceServiceClient, used ONLY for testing purposes.
+public class MockCloudTraceClient implements CloudTraceClient{
```

Comment:
This class could be package private I think

Reply:
Done